runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -197,8 +198,8 @@ private void rebuildAsync(boolean updated, boolean skillPaused, XpSnapshotSingle
 			actionsLeft.setText(htmlLabel("Actions: ", xpSnapshotSingle.getActionsRemainingToGoal()));
 
 			// Update progress bar
-			progressBar.setValue(xpSnapshotSingle.getSkillProgressToGoal());
-			progressBar.setCenterLabel(xpSnapshotSingle.getSkillProgressToGoal() + "%");
+			progressBar.setValue((int) xpSnapshotSingle.getSkillProgressToGoal());
+			progressBar.setCenterLabel(new DecimalFormat("0.00").format(xpSnapshotSingle.getSkillProgressToGoal()) + "%");
```

Comment:
you should probably use StackFormatter class to format this, or declare decimal format as constant at top

Reply:
I opted to declare the decimal format as a constant at the top.